### PR TITLE
Conduit: init parameters added to conduit init command

### PR DESCRIPTION
**Pull request by mciccarello**
## Summary

Added parameters (actually Cobra calls them flags) for user to set importer, exporter and processors and have them show up in the same yaml configuration file.

## Test Plan

Modifications to unit test to come (reason for the draft status)

### CODE REVIEW

File: cmd/conduit/main.go [hunk 1]
```diff
@@ -155,7 +162,37 @@ func runConduitInit(path string) error {
...
+		exporter = defaultExporter
+	} else {
+		exporter = twoSpaces + "name: " + exporterFlag + "\n" + twoSpaces + "config:"
+	}
+
+	var processors string
+	for _, processor := range processorList {
+		processors = processors + twoSpaces + "- "
+		processors = processors + "name: " + processor + "\n" + twoSpaces + twoSpaces + "config:\n"
+	}
```
Comment (winder) [inline cmd/conduit/main.go:190]: could you add:
* validation - make sure the requested plugin exists.
* use the metadata template to fill in a sample config.

There are some helpers to fetch the metadata:
https://github.com/algorand/indexer/blob/develop/conduit/pipeline/metadata.go

The `./conduit list ...` command shows this data being used:
https://github.com/algorand/indexer/blob/develop/cmd/conduit/internal/list/list.go

File: cmd/conduit/main.go [hunk 2]
```diff
@@ -14,6 +15,7 @@ import (
 	"github.com/algorand/indexer/cmd/conduit/internal/list"
 	"github.com/algorand/indexer/conduit"
 	"github.com/algorand/indexer/conduit/pipeline"
+
```
Comment (winder) [inline cmd/conduit/main.go:18]: the lint error is related to the import order in this file. It is fixed on develop.

Comment (mciccarello) [reply cmd/conduit/main.go:18]: Thanks, the error message was so helpful.

Comment (mciccarello) [reply cmd/conduit/main.go:18]: Actually I merged develop and it doesn't change anything about the import order as far as I can tell, and the problem persists.

**Pull request closed without merging**